lib/libutee/tee_api.c

```diff
@@ -33,7 +33,7 @@
 #include "tee_user_mem.h"
 #include "tee_api_private.h"
 
-static void *tee_api_instance_data;
+static const void *tee_api_instance_data;
```

Comment:
I wrote that comment before I learned that `TEE_SetInstanceData()` really should take a const pointer.